### PR TITLE
Update mergeClasses doc to explain combining griffel classes with custom classes

### PR DESCRIPTION
**Pull request by benkeen**
Small doc update to provide an explicit example of combining the generated scoped classes from Griffel with arbitrary space-delimited class strings. 

As per discussion here: https://github.com/microsoft/griffel/issues/424

### CODE REVIEW

File: apps/website/docs/react/api/merge-classes.md [hunk 1]
```diff
@@ -112,6 +112,25 @@ function App(props) {
...
+
+function App(props) {
+  const classes = useClasses();
+  const rootClass = mergeClasses(classes.root, "custom-global-class1 custom-global-class2");
+
+  return (
+     <div className={rootClass}>...</div>
+  )
+}
+```
```
Comment (layershifter) [inline apps/website/docs/react/api/merge-classes.md:132]: I just realized that we have already an example for this case above "Incorrect usages":

image

I suggest to modify that section and add above it:

```
### Merging plain CSS classes

Your component may accept classes via props or have static classes that you need applied to the same elements being styled by Griffel. For this, you can also use `mergeClasses`:
```

WDYT?

Comment (benkeen) [reply apps/website/docs/react/api/merge-classes.md:132]: Sounds good!

Comment (layershifter) [reply apps/website/docs/react/api/merge-classes.md:132]: @benkeen would you be able to do the change? 🐱

Comment (benkeen) [reply apps/website/docs/react/api/merge-classes.md:132]: Yup absolutely! Sorry - last night was busy. Will update it tonight.

Comment (benkeen) [reply apps/website/docs/react/api/merge-classes.md]: Updated. Definitely better, thanks!

**Review by layershifter — APPROVED** [review]

**Pull request merged**